content/protocols/sse.textile

```diff
@@ -0,0 +1,113 @@
+---
+title: SSE
+meta_description: "Ably provides support for Server-Sent Events (SSE). This is useful for where browser clients support SSE, and the use case does not require or support the resources used by the Ably client library SDK."
+---
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an "MQTT":/protocols/mqtt library, is impractical.
+
+HTTP streaming allows for a request from a client to be held by a server, allowing it to push data to the client without further requests. This, much like WebSockets, help avoid the overhead involved in normal HTTP requests. "Server-sent events":https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events (SSE) provide a thin layer on top of HTTP streaming. A common use of SSE is through the use of "the EventSource API":https://developer.mozilla.org/en-US/docs/Web/API/EventSource in all modern web browsers.
+
+It is subscribe-only: you can not interact with the channel, including to publish, enter presence, query the presence set, attach and detach from channels (without closing and re-opening the stream), or anything else.
+
+Customers who do not want to use a client library on platforms that support SSE, and only require simple subscribe-only streams, may choose to use SSE because it's an open standard, simple, and requires no SDKs on the client-side. HTTP Streaming may be considered on platforms without an SSE client. However, where possible, we strongly recommend the use of one of our "Realtime client libraries":https://ably.com/download, which provide "more features and higher reliability":/basics/use-ably, and the full use of our normal realtime messaging API.
+
+h2(#when-to-use). When to use the SSE adapter
+
+At Ably we believe in protocol interoperability, and whilst we believe our own native realtime protocol in a lot of cases offers the right balance between performance, portability and quality of service, we recognize that developers should be given the choice to choose the right protocol for their applications based on their own use case. As such, we offer support for SSE in addition to a number of other protocols.
```

Comment:
This paragraph has a bit of a marketing spin on it. Is it necessary or can we drop it do you think?